### PR TITLE
Attempted contribution

### PR DESCRIPTION
**Pull request by desafecto**
Just some minor changes

### CODE REVIEW

File: debug-python-pdb/debug-browser.robot [hunk 1]
```diff
@@ -1,11 +1,11 @@
 *** Settings ***
 Documentation   Opens the browser and searches Bing
-Library         RPA.Browser
+Library         RPA.Browser.Selenium
```
Comment (desafecto) [inline debug-python-pdb/debug-browser.robot:3]: A depreciation message appeared when executing.

File: debug-python-pdb/debug-browser.robot [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 Library         DebugLibrary.py
 Suite Teardown  Teardown Actions
 
 *** Variables ***
-${BING_URL}     https://bing.com
```
Comment (desafecto) [inline debug-python-pdb/debug-browser.robot:8]: Minor changes regarding formatting recommended: always use 4 blanks

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,387 @@
+# Mark Percival previous considerations
 */output
-*.log
\ No newline at end of file
+*.log
+
+# Created by https://www.toptal.com/developers/gitignore/api/python,linux,macos,windows,vscode,pycharm,emacs
```
Comment (desafecto) [inline .gitignore:5]: Very useful website.  :)

**Comment by mdp** [comment]
Hey @desafecto, thanks for the fixes! I merged most of them. The only one I didn't merge with .gitignore since we prefer to have people manage OS specific ignores with .git/info/exclude or ~/.gitignore - https://www.atlassian.com/git/tutorials/saving-changes/gitignore#global-git-ignore-rules

**Pull request merged**